test/libethereum/Block.cpp

```diff
@@ -40,11 +41,12 @@ BOOST_AUTO_TEST_CASE(bStructures)
 	BlockDetail details;
 	BlockSafeExceptions exeptions;
 
-	BOOST_REQUIRE(string(chat.name()).find("◌") != string::npos);
-	BOOST_REQUIRE(string(trace.name()).find("◎") != string::npos);
-	BOOST_REQUIRE(string(details.name()).find("◌") != string::npos);
-	BOOST_REQUIRE(string(exeptions.name()).find("ℹ") != string::npos);
+	BOOST_WARN(string(chat.name()).find("◌") != string::npos);
+	BOOST_WARN(string(trace.name()).find("◎") != string::npos);
+	BOOST_WARN(string(details.name()).find("◌") != string::npos);
```

Comment:
Please search the source for something like `#if defined(WIN32)` and use exactly what is used in other places.